specification/migrate/resource-manager/Microsoft.Migrate/stable/2018-02-02/migrate.json

```diff
@@ -1670,9 +1709,183 @@
           }
         }
       }
+    },
+    "AssessmentOptionsResultList": {
+      "type": "object",
+      "description": "List of assessment options.",
+      "properties": {
+        "vmFamilies": {
+          "type": "array",
+          "description": "Dictionary of VM families grouped by vm family name describing the targeted azure locations of VM family and the category of the family.",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/VmFamily"
+          }
+        },
+        "reservedInstanceVmFamilies": {
+          "type": "array",
+          "description": "List of supported VM Families.",
+          "readOnly": true,
+          "items": {
+            "type": "string",
+            "description": "Azure VM family name.",
+            "readOnly": true
+          }
+        }
+      }
     }
   },
   "paths": {
+    "/subscriptions/{subscriptionId}/deleteSubscriptionNotification": {
+      "delete": {
+        "summary": "Delete all projects in the subscription.",
+        "description": "Delete all projects in the subscription. Deleting non-existent project is a no-operation.",
+        "operationId": "SubscriptionNotification_Delete",
```

Comment:
Given the description of this operation - I would probably change the operation ID to something like "Projects_DeleteBySubscription" or "Projects_DeleteAll".
But you better understand the chosen naming convention, so up to you :)

Reply:
Done.